Userland/DevTools/HackStudio/HackStudioWidget.cpp

```diff
@@ -647,6 +648,18 @@ NonnullRefPtr<GUI::Action> HackStudioWidget::create_new_project_action()
     });
 }
 
+NonnullRefPtr<GUI::Action> HackStudioWidget::create_remove_current_editor_tab_widget_action()
+{
+    return GUI::Action::create("Switch to Next Editor Group", { Mod_Alt | Mod_Shift, Key_Backslash }, [this](auto&) {
+        if (m_all_editor_tab_widgets.size() <= 1)
+            return;
+        auto tab_widget = m_current_editor_tab_widget;
+        m_switch_to_next_editor_tab_widget->activate();
+        m_editors_splitter->remove_child(*tab_widget);
+        m_all_editor_tab_widgets.remove_first_matching([&tab_widget](auto& entry) { return entry == tab_widget.ptr(); });
```

Comment:
I also need to iterate through all of the editor wrappers contained inside this tab widget and remove them from the `m_all_editor_wrappers` vector. Not doing so will affect the logic in the rest of the code and crash.